packages/squiggle-lang/src/rescript/ReducerInterface/ReducerInterface_GenericDistribution.res

```diff
@@ -240,6 +258,34 @@ let dispatchToGenericOutput = (
   | ("normalize", [EvDistribution(dist)]) => Helpers.toDistFn(Normalize, dist, ~env)
   | ("klDivergence", [EvDistribution(a), EvDistribution(b)]) =>
     Some(DistributionOperation.run(FromDist(ToScore(KLDivergence(b)), a), ~env))
+  | (
+    "logScoreWithPointResolution",
```

Comment:
It's weird that you call it "resolution" here, but "answer" in the record view. I'd probably rename this to "answer"; like, "logScoreWithPointAnswer"